python-threatexchange/threatexchange/signal_type/index.py

```diff
@@ -48,6 +48,19 @@ def __init__(self, distance: int, metadata: T) -> None:
         self.metadata = metadata
 
 
+class VPDQIndexMatch(IndexMatch):
+    query_match_percent: float
+    compared_match_percent: float
+
+    def __init__(
+        self, query_match_percent: float, compared_match_percent: float, metadata: T
+    ) -> None:
+        # Dist is meaning less here, because VPDQ match does not return a single dist
+        IndexMatch.__init__(self, -1, metadata)
```

Comment:
blocking: You should never call the superclass like this, instead always use super().

The reason for this is that in multiple inheritence, or if someone refactors in a middle layer, this will break the expected behavior of initialization, and possibly be very hard to discover